### ISSUE TITLE
Help needed to finetune the Pretrain model of hindi on custom dataset.

### ISSUE THREAD

**Issue by shekharmeena2896** [SWivid/F5-TTS#1045]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

Hi,
I have been trying to finetune the model of hindi which is provided in the community and also in the docs, I followed every step of the data preparation.
But I encountered an error while I was fine-tuning the model. this  below what is I obtained there is no output 

this is the config of Hindi pre trained model:

(f5tts) root@t1-le-45-gra7:/home/ubuntu/F5-TTS/F5-Hindi-24KHz# cat config.yaml 
model:
  name: F5TTS_Base
  tokenizer: pinyin
  tokenizer_path: null
  backbone: DiT
  arch:
    dim: 768
    depth: 18
    heads: 12
    ff_mult: 2
    text_dim: 512
    text_mask_padding: false
    conv_layers: 4
    pe_attn_head: 1
    checkpoint_activations: false
  mel_spec:
    target_sample_rate: 24000
    n_mel_channels: 100
    hop_length: 256
    win_length: 1024
    n_fft: 1024
    mel_spec_type: vocos
  vocoder:
    is_local: false
    local_path: null

(f5tts) root@t1-le-45-gra7:/home/ubuntu/F5-TTS/ckpts# accelerate launch   /home/ubuntu/F5-TTS/src/f5_tts/train/finetune_cli.py   --exp_name      F5TTS_Base   --learning_rate 1e-5   --batch_size_per_gpu 3200   --batch_size_type frame   --max_samples 64   --grad_accumulation_steps 1   --max_grad_norm 1.0   --epochs        50   --dataset_name  hindi_dataset   --finetune   --pretrain      "/home/ubuntu/F5-TTS/F5-Hindi-24KHz/model_2500000.pt"   --tokenizer_path "/home/ubuntu/F5-TTS/F5-Hindi-24KHz/vocab.txt"   --tokenizer     custom

vocab :  105

vocoder :  vocos
Using logger: None
Loading dataset ...
Sorting with sampler... if slow, check whether dataset is provided with duration: 100%|██████████████████████████████████████████████████████| 4700/4700 [00:00<00:00, 1502647.21it/s]
Creating dynamic batches with 3200 audio frames per gpu: 100%|███████████████████████████████████████████████████████████████████████████████| 4700/4700 [00:00<00:00, 1737613.82it/s]
/root/anaconda3/envs/f5tts/lib/python3.10/site-packages/torch/utils/data/dataloader.py:626: UserWarning: This DataLoader will create 16 worker processes in total. Our suggested max number of worker in current system is 8, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(
Saved last checkpoint at update 2500000
(f5tts) root@t1-le-45-gra7:/home/ubuntu/F5-TTS/ckpts#  

I don't now why it is executing fast, I am using the finetuning_cli.py to run the script and I have dataset in the format  and also I am confused with test_train.py, what should I changed about It , it does not exist in the repo.

**Comment by SWivid** [comment]
Not quite get your question, there are some points need clarification:
1. `config.yaml` you use yaml which is configuration for `train.py`, but your command is `finetune_cli.py` seems with `finetune_gradio.py` which is configured with ui and dumped to json file. These two things are not related.
2. the `config.yaml` seems incomplete, where is other configs e.g. checkpoint saving related stuff?
3. > I am using the finetuning_cli.py to run the script and I have dataset in the format and also I am confused with test_train.py
not sure about the `test_train.py` here, and are you using `finetuning_cli.py` but not infer-gradio?

Some suggestions:
1. Go though discussion board and issue mentioned in [training README.md](https://github.com/SWivid/F5-TTS/tree/main/src/f5_tts/train) before start. There are also many valuable discussions not listed but cannot be found with proper searching.
2. Use a help-wanted template. The current template is `only for research question, not usage problems, feature requests or bug reports.`
